docs/config-schema.md

```diff
@@ -54,7 +54,7 @@ if not set.
 | `writer` | no | [object (see below)](#writer) | Configuration of the datapoint/event writer |
 | `logging` | no | [object (see below)](#logging) | Log configuration |
 | `collectd` | no | [object (see below)](#collectd) | Configuration of the managed collectd subprocess |
-| `enableBuiltInFiltering` | no | bool | If true, the agent will inherently filter non-bundled metrics without having to rely on the `whitelist.json` filter that was previously included under `metricsToExclude`.  Whether a metric is bundled or not is documented in each monitor's documentation.  Now every monitor's default configuration (i.e. the minimum amount of configuration to make it work) will only send bundled metrics, but if you enable certain flags or config options on the monitor, non-bundled metrics will also be sent without having to modify the whitelist via `metricsToInclude`. (**default:** `false`) |
+| `enableBuiltInFiltering` | no | bool | If true, the agent will inherently filter out [custom metrics](https://docs.signalfx.com/en/latest/admin-guide/usage.html#about-custom-bundled-and-high-resolution-metrics) without having to rely on the `whitelist.json` filter that was previously configured under `metricsToExclude`.  Whether a metric is custom or not is documented in each monitor's documentation.  Now every monitor's default configuration (i.e. the minimum amount of configuration to make it work) will only send non-custom metrics, but if you enable certain flags or config options on the monitor, custom metrics will also be sent without having to modify the whitelist via `metricsToInclude`.  If this option is set to `true`, the `whitelist.json` entry under `metricToExclude` should be removed, if it was present. (**default:** `false`) |
```

Comment:
I can't quite figure out what this is saying. Either it's saying the same thing different ways, or it's mushing together a few different ideas, or something.. Especially the sentence starting "Now every monitor's.."  You mention metricsToInclude and metricsToExclude and it's just a bit unclear. Seems like it needs to be broken out into a couple paragraphs or bullet list, maybe. 

And if I'm reading it right, it sounds like this option replaces the need for whitelist.json and metricsToExclude. If I'm reading it right, you should say it in that way. Also, in the docs for whitelist.json and metricsToExclude, are you going to mention that that usage is deprecated or superseded in favor of this usage?  (Or just delete it altogether and refer to this option instead?) 

Also, please make the last sentence active: "If you set this option to true, delete the whitelist.json entry..."  BTW, what happens if they don't delete it?

Reply:
Ok, I tried to clarify it a bit.  See if it makes more sense to you now.